core/ui/compose/theme2/k9mail/src/main/kotlin/app/k9mail/core/ui/compose/theme2/k9mail/K9MailTheme2.kt

```diff
@@ -0,0 +1,48 @@
+package app.k9mail.core.ui.compose.theme2.k9mail
+
+import androidx.compose.foundation.isSystemInDarkTheme
+import androidx.compose.runtime.Composable
+import app.k9mail.core.ui.compose.theme2.MainTheme
+import app.k9mail.core.ui.compose.theme2.ThemeColorSchemeVariants
+import app.k9mail.core.ui.compose.theme2.ThemeConfig
+import app.k9mail.core.ui.compose.theme2.ThemeImageVariants
+import app.k9mail.core.ui.compose.theme2.ThemeImages
+import app.k9mail.core.ui.compose.theme2.default.defaultThemeElevations
+import app.k9mail.core.ui.compose.theme2.default.defaultThemeShapes
+import app.k9mail.core.ui.compose.theme2.default.defaultThemeSizes
+import app.k9mail.core.ui.compose.theme2.default.defaultThemeSpacings
+import app.k9mail.core.ui.compose.theme2.default.defaultTypography
+
+@Composable
+fun K9MailTheme2(
```

Comment:
We typically only use `K9` as prefix, not `K9Mail`.

Reply:
What about writing that down? I see both in the project.